Sources/Knit/Module/ModuleAssembly.swift

```diff
@@ -40,9 +40,8 @@ public extension ModuleAssembly {
     static var replaces: [any ModuleAssembly.Type] { [] }
 
     static func scoped(_ dependencies: [any ModuleAssembly.Type]) -> [any ModuleAssembly.Type] {
```

Comment:
Can we remove `static func scoped(` from the public protocol now? Seems like it is no longer the extension point